fenixedu-ist-integration/src/main/java/pt/ist/fenixedu/integration/dto/PersonInformationDTO.java

```diff
@@ -177,6 +180,13 @@ public PersonInformationDTO(final Person person) {
             final Registration lastActiveRegistration = person.getStudent().getLastActiveRegistration();
             if (lastActiveRegistration != null) {
                 this.campus = lastActiveRegistration.getCampus().getName();
+            } else if (new ActivePhdProcessesGroup().isMember(person.getUser())) {
+                Optional<PhdIndividualProgramProcess> individualProgramProcess =
+                        person.getPhdIndividualProgramProcessesSet().stream().findFirst();
```

Comment:
it can not be the first one of the list, the person may have more than one phd process, it has to be the first one of the active phd processes